crypto/src/value.rs

```diff
@@ -298,7 +298,7 @@ mod tests {
     #[test]
     fn format_picks_best_unit() {
         let upenumbra_base_denom = asset::REGISTRY.parse_denom("upenumbra").unwrap();
-        let gm_base_denom = asset::REGISTRY.parse_denom("gm").unwrap();
+        let gm_base_denom = asset::REGISTRY.parse_denom("ugm").unwrap();
```

Comment:
it doesn't look like the `gm_base_denom` is ever used after this? should there be another test case? (not sure it'd really exercise anything different than the existing cases though)